README.md

```diff
@@ -256,8 +256,7 @@ after an LGTM comment is in the [scala/scabot](https://github.com/scala/scabot)
 The Scala community build is an important method for testing Scala
 releases. A community build can be launched for any Scala commit, even
 before the commit's PR has been merged. That commit is then used to
-build a large number of open-source projects from source and run their
-test suites.
+build many open-source projects from source and run their test suites.
```

Comment:
IMO the original is more correct. Emphasis is "a large number", not "many [of some number, but not all]". We can say "a few projects" but not "a many projects", hence this locution.